### ISSUE TITLE
Version 2.3.4 pasting doesn't work on linux

### ISSUE THREAD

**Issue by Mikaka27** [gazugafan/vscode-indent-on-paste#12]
I updated this extension to version 2.3.4 and ever since then it stopped working, nothing was ever pasted, but I noticed this only happens on Linux, Windows is fine (no access to Mac).

When I looked into developer tools I saw this problem:
```
[Extension Host] Error pasting... Error: Couldn't find the required `xsel` binary. On Debian/Ubuntu you can install it with: sudo apt install xsel
    at handler (/home/mikaka/.vscode/extensions/gazugafan.vscode-indent-on-paste-2.3.4/node_modules/clipboardy/lib/linux.js:7:9)
    at Object.pasteSync (/home/mikaka/.vscode/extensions/gazugafan.vscode-indent-on-paste-2.3.4/node_modules/clipboardy/lib/linux.js:44:5)
    at Object.exports.readSync (/home/mikaka/.vscode/extensions/gazugafan.vscode-indent-on-paste-2.3.4/node_modules/clipboardy/index.js:41:37)
    at getClipboard (/home/mikaka/.vscode/extensions/gazugafan.vscode-indent-on-paste-2.3.4/out/src/extension.js:119:23)
    at indentOnPaste (/home/mikaka/.vscode/extensions/gazugafan.vscode-indent-on-paste-2.3.4/out/src/extension.js:14:27)
    at /snap/code/33/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:897:216
    at t.ExtHostTextEditor.edit (/snap/code/33/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:584:588)
    at /snap/code/33/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:897:205
    at e._executeContributedCommand (/snap/code/33/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:627:894)
    at e.$executeContributedCommand (/snap/code/33/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:628:295)
    at e._doInvokeHandler (/snap/code/33/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:788:757)
    at e._invokeHandler (/snap/code/33/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:788:449)
    at e._receiveRequest (/snap/code/33/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:787:120)
    at e._receiveOneMessage (/snap/code/33/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:785:970)
    at /snap/code/33/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:784:213
    at e.fire (/snap/code/33/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:46:67)
    at v.fire (/snap/code/33/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:232:615)
    at /snap/code/33/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:957:179
    at e.fire (/snap/code/33/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:46:67)
    at v.fire (/snap/code/33/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:232:615)
    at t.PersistentProtocol._receiveMessage (/snap/code/33/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:237:17)
    at /snap/code/33/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:234:155
    at e.fire (/snap/code/33/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:46:67)
    at p.acceptChunk (/snap/code/33/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:230:129)
    at /snap/code/33/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:229:483
    at Socket.t (/snap/code/33/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:238:565)
    at Socket.emit (events.js:203:13)
    at addChunk (_stream_readable.js:295:12)
    at readableAddChunk (_stream_readable.js:276:11)
    at Socket.Readable.push (_stream_readable.js:210:10)
    at Pipe.onStreamRead (internal/stream_base_commons.js:166:17)
```

Anyway I have vscode installed via Microsoft's official snap image, so I cannot do an apt-get inside of it.
Also when I'm at work, we use vscode prepackaged by IT departament and I see the same problem there, pasting also does not work.

When I downgrade this extension to version 2.3.3 it works as expected.

**Comment by gazugafan** [comment]
Thanks for the report. Not sure what could be wrong here, unfortunately. xsel is included with clipboardy, which this extension depends on. v2.3.4 is literally a one-line change, and shouldn't have anything to do with xsel being included or not. I didn't update clipboardy with this release, so nothing should be changed there. I can confirm that both v2.3.3 and v2.3.4 have xsel in node_modules\clipboardy\fallbacks\linux. Do you see it there, too?

**Comment by Mikaka27** [comment]
Hi,
Yes I see the xsel there, however I noticed one difference.
This is on version 2.3.4:
```
mikaka@Stacjonarny:~/.vscode/extensions/gazugafan.vscode-indent-on-paste-2.3.4/node_modules/clipboardy/fallbacks/linux$ ls -lah
razem 136K
drwxrwxr-x 2 mikaka mikaka 4,0K maj 28 14:22 .
drwxrwxr-x 4 mikaka mikaka 4,0K maj 28 14:22 ..
-rw-rw-r-- 1 mikaka mikaka 127K maj 28 14:22 xsel
```
and this on 2.3.3:
```
mikaka@Stacjonarny:~/.vscode/extensions/gazugafan.vscode-indent-on-paste-2.3.3/node_modules/clipboardy/fallbacks/linux$ ls -lah
razem 136K
drwxrwxr-x 2 mikaka mikaka 4,0K maj 29 09:07 .
drwxrwxr-x 4 mikaka mikaka 4,0K maj 29 09:07 ..
-rwxr-xr-x 1 mikaka mikaka 127K maj 29 09:07 xsel
```

The xsel binary doesn't have exectuable bit set on 2.3.4

I see the same thing on our linux machine at work, it also doesn't have executable bit on 2.3.4 but does on 2.3.3


If I manually do `chmod u+x xsel` pasting starts to work normally.

**Comment by gazugafan** [comment]
GAAHH.... I completely forgot about this. Same as issue #3  ... I need to publish the extension from a Linux machine for the executable permission to stay intact. Will try to get a new version out later today.

**Comment by gazugafan** [comment]
Just released v2.3.5. Should be working now! Made a note to myself in the README, so hopefully I don't forget to publish from Linux next time 😅
